src/style.css

```diff
@@ -1,3 +1,178 @@
-.hello {
-  color: red;
+html,
+body {
+  width: 100%;
+  height: 100%;
+  padding: 0%;
+  margin: 0%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.list-title {
+  width: 100%;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: monospace;
+  font-size: 18px;
+  font-weight: 600;
+  border-bottom: 1px solid grey;
+}
+
+.list-title > * {
+  margin: 0% 8px;
+}
+
+.undo {
+  background-color: unset;
+  padding: 0%;
+  border: none;
+  outline: none;
+}
+
+.template {
+  max-width: 400px;
+  width: auto;
+  min-width: 280px;
```

Comment:
[ Optional Change ]

- It would be nice if you add the width of the Todo list more. It looks too small and since it's a web app, please don't hesitate to make it look big a little. 600px min-with will be enough. 👍